### ISSUE TITLE
Pruning: `keepblocks` value should cover a 15-day auction period

### ISSUE THREAD

**Issue by pinheadmz** [handshake-org/hsd#644]
Pruning mode in hsd means only the last 288 blocks are kept on disk:

https://github.com/handshake-org/hsd/blob/d393c5a4ac4f65fc6230a37ba24ed1ee9dd26a02/lib/protocol/networks.js#L394-L398

We should consider bumping this up to one complete auction cycle so that "rescan auction" is still available in pruned mode.

**Comment by nodech** [comment]
Does that mean it would be: `bidding period` + `reveal period` ?
```
That would mean:
  blocks per day - 144
  bidding period: 5 days - 144 * 5 - 720 blocks
  reveal period: 10 days - 144 * 10 - 1440 blocks
  total: 2160 blocks?
```

**Comment by pinheadmz** [comment]
@nodech yup, seems like a ton huh. I guess pruning node operators should already know they can't rescan their wallet, and the `rescan auction` function is the only other thing that will break on pruned nodes. However, I *do* want Bob Wallet to compete with Namebase's UI which can show a user all auction activity at any arbitrary time, and I *also* want a pruning mode available in Bob to reduce the burden on full node users... I'm not sure what the compromise should be. Maybe we can reduce it to 5 days, just the bidding period. Rescanning an auction once REVEAL starts doesn't really do you any good if you're not already "watching" the auction.
